test/ctia/entity/relationship_test.clj

```diff
@@ -172,11 +178,84 @@
 
          (is (= (:id casebook-body)
                 (:source_ref link-response))
+             "The New Relationship sources the casebook")
+
+         (is (= (:id incident-body)
+                (:target_ref link-response))
              "The New Relationship targets the incident")
+         (is (= relationship-response
+                link-response)
+             "Link Response is the created relationship"))))))
+
+(deftest incident-investigation-link-routes-test
+  (test-for-each-store
+   (fn []
+     (helpers/set-capabilities! "foouser" ["foogroup"] "user" all-capabilities)
+     (whoami-helpers/set-whoami-response "45c1f5e3f05d0"
+                                         "foouser"
+                                         "foogroup"
+                                         "user")
+     (testing "Incident <-> Investigation link"
```

Comment:
The `testing` block is quite large, but in this case it seems justified since it's a stateful test that "tells a story" out of multiple requests.

Maybe expand `"Incident <-> Investigation link"` to summarize that story, else one finds himself reading a large `let` without knowing exactly what to expect.